### ISSUE TITLE
Conveyer Lifts : "F" doesn't flip the input and output but rotates it 180 degrees

### ISSUE THREAD

**Issue by BrickedKeyboard** [MaFi-Games/Captain-of-Industry-issues#1387]
### Issue description

Update 2, you frequently have a conveyor lift already at the height you want, such as on the supply for the transport.  "F" should be flipping the input and output arrows so you can copy paste the lift to reuse it on the output for the transport, not rotating 180 degrees which is basically useless.

It's so obviously wrong I'm reporting it as a bug, hopefully its a trivial fix.

### Reproduction steps

Launch update 2.
Put down a conveyer lift configured for a height.
Copy it and try to use it at the destination, you can't, got to start over.

### Game version and build number

b314

### Operating system

Windows

**Comment by Unibozu** [comment]
Adding some input to this bug report as I had this happening just now.

It happens when you hover over a ghost item and hit "F" which flips the direction of the transport but also rotate the ghost. Move your move out of the way and it works as intended.

**Comment by zuff-coi** [comment]
Yes this is a tricky one. Like @Unibozu said, selecting the lift after it is placed and moving away your mouse allows "F" to reverse the direction. When your mouse is over any ghost object in the game "F" flips the object. 

I'm sure there is a better way we can handle this but currently it is working as intended.
